project_files/Xcode/hector.xcodeproj/project.pbxproj

```diff
@@ -7,8 +7,6 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		270FA7802791BF2F00E6A50B /* nh3_component.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 270FA77F2791BF2F00E6A50B /* nh3_component.cpp */; };
-		270FA7812791BF2F00E6A50B /* nh3_component.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 270FA77F2791BF2F00E6A50B /* nh3_component.cpp */; };
```

Comment:
Is there a reason why the nh3_componet is being removed? Will this cause issues?

Reply:
well good news is that I was able to check out this branch and build & run via Xcode

Reply:
OK that's good. I see nh3_component.cpp so not sure why the deletion above.